.github/workflows/send-content-to-translate.yml

```diff
@@ -0,0 +1,39 @@
+name: Send content to be translated
+
+on:
+  schedule:
```

Comment:
We should add the ability to manually trigger this.

Reply:
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
We also have this setup for the sync action: https://github.com/newrelic/docs-website/blob/develop/.github/workflows/sync-data-dictionary.yml#L4